files/zuul-install-requirements-pip.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
This playbook is 100% subset of the previous one, can we merge and parametrize them?

Reply:
fixed